src/services/output/confirm.ts

```diff
@@ -175,7 +175,7 @@ export class Confirm extends Prompt {
     output.updatePrompt(
       sprintln({
         ...this.options,
-        content: `${this.options.content} ${this.defaultValue ? "[Y/n] " : "[y/N] "}`,
+        content: `${this.options.content} ${this.defaultValue ? chalk.blueBright("[Y/n]") : chalk.blueBright("[y/N]")}`,
```

Comment:
Did we not need the space after the yes/no label?

Reply:
I didn't think so on first glance, but we don't have any tests to double check that. Just to be on the safe side I'll add it back in.